website/docs/r/group.html.markdown

```diff
@@ -65,6 +71,10 @@ The resource exports the following attributes:
 
 * `runners_token` - The group level registration token to use during runner setup.
 
+* `subgroup_creation_level` - Who can create subgroup
```

Comment:
Input arguments should not be repeated as attributes.

Reply:
fixed